src/scss/components/_menu.scss

```diff
@@ -27,7 +27,7 @@
   &--over {
     width: 100%;
     position: fixed;
-    z-index: 2;
+    z-index: 10;
```

Comment:
@MarcoBrunoBR @GeorgeJegon @eduardojmatos If the use of `z-index` start to increase too much, we have to give more attention to it and create some variables to organize it better.